decentraland/unity-renderer
# feat: channels window implementation & flow wiring

## What does this PR change?

Adds a new view/controller to the channels window.
Wire up the navigation for specific channels.
#nearby channel keeps its special controller.

## How to test the changes?

1. Open the conversation list window
2. Click on a channel
3. A window should be opened showing messages with that specific channel

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
